### DISCUSSION TITLE
Nueva página Formas de entrega

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#47] [General]
Con la nueva versión de la página de formas de entrega se integra la configuración de los mensajeros, permitiendo optimizar los tiempos de configuración, los procesos operativos y facilitando el uso del sistema.

**Comment by NestorFGG** [comment]
Optimización de opciones, se unen para facilitar la configuración las páginas de Formas de entrega y mensajeros.

**Comment by NestorFGG** [comment]
Nueva página con implementación de Diseño

**Comment by aleja550** [comment]
Nunca se realizo ni tampoco está en planeación del producto actual.
